packages/server/lib/modes/run-e2e.ts

```diff
@@ -0,0 +1,5 @@
+import type { LaunchArgs } from '@packages/types'
+
+export const run = (options: LaunchArgs, loading: Promise<void>) => {
+  return require('./run').run(options, loading)
```

Comment:
I wonder if we want to start moving towards ESM more?

```js
import { run as runE2E } from './run'

export const run = (options: LaunchArgs, loading: Promise<void>) => {
  return runE2E(options, loading)
}
```

Reply:
🤷🏻‍♀️ I don't have a strong pref, just copied run-ct 😄 https://github.com/cypress-io/cypress/blob/clean-up-server/packages/server/lib/modes/run-ct.ts